src/tour/TourManager.ts

```diff
@@ -164,18 +164,28 @@ export default class TourManager {
   }
 
   private setHighlight(mask: IBoundingBox) {
+    const fullAppHeight = this.backdrop.ownerDocument.body.scrollHeight; // full page height including scroll bars
+    const fullAppWidth = this.backdrop.ownerDocument.body.scrollWidth; // full page width including scroll bars
+
+    // set the new height of the backdrop
+    this.backdrop.style.height = `${fullAppHeight}px`;
+    this.backdrop.style.width = `${fullAppWidth}px`;
+
+    // also consider the current scroll offset inside the window
+    const scrollOffset = self.scrollY;
```

Comment:
`scrollY` is a property of which object?

Reply:
the window